core/components/molecules/modal/__stories__/Dialog.story.jsx

```diff
@@ -122,7 +123,7 @@ const customCode = `() => {
             description="Following 11 outreaches are currently scheduled using this sender's details."
           />
         </div>
-        <div style={{ border: 'var(--border)', height:'185px' }} className="my-5 overflow-auto">
+        <div className="my-5 overflow-auto border Modal-list">
```

Comment:
Please add CSS content as a comment in the custom code to share the style with the end user. [Refer this example](https://github.com/innovaccer/design-system/blob/master/core/components/patterns/table/TableWithHeader/tableWithHeader.story.jsx#L30)